### PR TITLE
added cl4-5

### PR DESCRIPTION
**Pull request by Fatih-Gun**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.idea/
+out/
```
Comment (Ayse-Gun) [inline .gitignore:2]: need to work

File: src/test/java/class4.java [hunk 2]
```diff
@@ -0,0 +1,2 @@
+public class class4 {
```
Comment (Ayse-Gun) [inline src/test/java/class4.java:1]: change its name

Comment (Fatih-Gun) [reply src/test/java/class4.java:1]: I changed it

**Review by Ayse-Gun — CHANGES_REQUESTED** [review]
need to work

File: .gitignore (hunk 1 above)
Comment (Fatih-Gun) [reply .gitignore:2]: ok

File: src/test/java/class4.java (hunk 2 above)
Comment (Fatih-Gun) [reply src/test/java/class4.java:1]: ok

**Review by Fatih-Gun — COMMENTED** [review]
igtm

**Review by Ayse-Gun — APPROVED** [review]

**Pull request merged**